src/components/Accordion/Accordion.types.ts

```diff
@@ -23,6 +23,36 @@ export interface AccordionProps extends AccordionBaseProps {
      * Accordion Summary
      */
     summary: React.ReactNode | string;
+    /**
+     * Header Classnames
+     * @default ''
+     */
+    headerClasses?: string;
+    /**
+     * Body Classnames
+     * @default ''
+     */
+    bodyClasses?: string;
+    /**
+     * Header Styles
+     * @default {}
+     */
+    headerStyles?: Object;
```

Comment:
can we instead change this to, and omit anything you dont need 
```suggestion
    headerProps?: AccordionSummaryProps;
    bodyProps?: AccordionBodyProps;
```